web-ui/src/streaming/PipelineTable.tsx

```diff
@@ -389,7 +383,11 @@ export default function PipelineTable() {
 
   const startPipelineClick = useCallback(
     (curRow: PipelineDescr) => {
-      if (!pipelineActionLoading && isLaunching.get(curRow.pipeline_id) != ClientPipelineStatus.PAUSED) {
+      if (
+        !pipelineActionLoading &&
+        (isLaunching.get(curRow.pipeline_id) != ClientPipelineStatus.PAUSED ||
```

Comment:
[ClientPipelineStatus.PASSED, ...].includes(isLaunching.get(...)) evaluates the get only once.

Reply:
Ok will change in next PR when I'm using leonids new APIs